examples/partytown/tsconfig.json

```diff
@@ -1,23 +1,16 @@
 {
+  "extends": "../../templates/skeleton/tsconfig.json",
   "include": ["./**/*.d.ts", "./**/*.ts", "./**/*.tsx"],
   "compilerOptions": {
-    "lib": ["DOM", "DOM.Iterable", "ES2022"],
-    "isolatedModules": true,
-    "esModuleInterop": true,
-    "jsx": "react-jsx",
-    "moduleResolution": "Bundler",
-    "resolveJsonModule": true,
-    "module": "ES2022",
-    "target": "ES2022",
-    "strict": true,
-    "allowJs": true,
-    "forceConsistentCasingInFileNames": true,
-    "skipLibCheck": true,
     "baseUrl": ".",
-    "types": ["@shopify/oxygen-workers-types"],
     "paths": {
-      "~/*": ["app/*"]
+      "*": ["./*", "../../templates/skeleton/*"],
+      "~/*": ["app/*", "../../templates/skeleton/app/*"]
     },
-    "noEmit": true
+    /***********************************************/
+    /**********  EXAMPLE UPDATE STARTS  ************/
+    "types": ["@shopify/oxygen-workers-types", "@types/gtag.js"]
```

Comment:
I just noticed this part. We've never added extra types to a diff example in tsconfig.json so we don't support it yet when scaffolding the template. We only use the skeleton's tsconfig.json for that.

If you have some energy left to work on this, [here](https://github.com/Shopify/hydrogen/blob/main/packages/cli/src/lib/template-diff.ts#L244) is the place where we skip copying the diff's tsconfig.json to the generated folder. That way, we keep the skeleton's tsconfig.json which is the correct thing to do.

We'd need to "merge" tsconfigs below that like we merge package.jsons already. We could probably do something like the following (haven't tried the code):

```ts
const allTypes = /("@shopify\/oxygen-workers-types"[^\]]*)\]/.match(diffTsconfigString)?.[1];

if (allTypes) {
  generatedTsconfigString = generatedTsconfigString.replace(/"@shopify\/oxygen-workers-types"/, allTypes)
}
```